src/modules/windowwalker/app/Window Walker/Components/Window.cs

```diff
@@ -215,7 +215,7 @@ public bool IsToolWindow
         }
 
         /// <summary>
-        /// Get a value indicating whether the window GWL_EX_STYLE is an appwindow
+        /// Gets a value indicating whether get a value indicating whether the window GWL_EX_STYLE is an appwindow

```

Comment:
There are three more places where `Gets a value indicating whether get a value` is repeated